nss-day-cohort-47/automated-world-surprising-spuds
# linked ireland image to index page

# Description
Linked 2nd image to index page
Fixes # Image Link
## Type of change
Please delete options that are not relevant.


# Testing Instructions
Fetch dcu-ireland-update, serve from it, see image on index page under Ireland section
# Checklist:
